### PR TITLE
update to Ionic v7 & angular v16

### PR DESCRIPTION
**Pull request by qhoirulanwar**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,22 +31,22 @@
...
-    "@angular-eslint/eslint-plugin": "14.1.2",
-    "@angular-eslint/eslint-plugin-template": "14.1.2",
-    "@angular-eslint/template-parser": "14.1.2",
-    "@angular/cli": "14.2.4",
-    "@angular/compiler": "14.2.4",
-    "@angular/compiler-cli": "14.2.4",
-    "@angular/language-service": "14.2.4",
+    "@angular-devkit/architect": "0.1601.4",
+    "@angular-devkit/build-angular": "16.1.4",
+    "@angular-eslint/builder": "^16.0.0",
```
Comment (robingenz) [inline package.json]: Please keep the exact version numbers.

**Review by robingenz — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (qhoirulanwar) [reply package.json]: I have done the update, please check again

File: package.json [hunk 2]
```diff
@@ -31,22 +31,22 @@
...
+    "@angular-eslint/builder": "16.0.3",
+    "@angular-eslint/eslint-plugin": "16.0.3",
+    "@angular-eslint/eslint-plugin-template": "16.0.3",
+    "@angular-eslint/template-parser": "16.0.3",
+    "@angular/cli": "16.1.4",
+    "@angular/compiler": "16.1.4",
+    "@angular/compiler-cli": "16.1.4",
+    "@angular/language-service": "16.1.4",
     "@capacitor/cli": "5.0.0",
     "@ionic/angular-toolkit": "7.0.0",
```
Comment (robingenz) [inline package.json:51]: Please update to `10.0.0`.

**Review by robingenz — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (qhoirulanwar) [reply package.json:51]: what package name do i need to change to version 10 ?

Comment (robingenz) [reply package.json:51]: `@ionic/angular-toolkit`

**Pull request closed without merging**